src/screens/LinkContextScreen/index.js

```diff
@@ -175,7 +175,7 @@ const styles = StyleSheet.create({
     alignSelf: 'center'
   },
   bottomAnchorTextContainer: {
-    backgroundColor: COLORS.bondi_blue,
+    backgroundColor: COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.bondi_blue` to `COLORS.anon_primary` seems fine as long as the new color `anon_primary` is defined in the `COLORS` object and it fits well with the overall theme of the application. Please ensure that this color change doesn't affect the readability or user experience negatively.
```diff
-    backgroundColor: COLORS.bondi_blue,
+    backgroundColor: COLORS.anon_primary,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->